packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -841,15 +841,23 @@ class _StretchController extends ChangeNotifier {
 
   double get value => _stretchSize.value;
 
+  // Constants for absorbImpact.
+  static const double _minVelocity = 1;
+  static const double _maxVelocity = 10000;
+  static const Duration _minStretchDuration = Duration(milliseconds: 50);
+
```

Comment:
@Piinks `Google testing` step failed again so I thought I'd update the branch in #146280 with the latest `master`.

I managed to ruin the commit tree so I've opened this PR instead. 

Please have a look at the constants introduced here as per the [feedback](https://github.com/flutter/flutter/pull/146280#pullrequestreview-2007035756) left by @chunhtai

Reply:
for const, you should start with `_k`